oauth2/provider/login.php

```diff
@@ -66,11 +69,32 @@
                         </div>
                         <?php } ?>
 
-                        <?php if (empty($authorize) && $mfaRequired && $mfaType === 'TOTP') { ?>
+                        <?php if (empty($authorize) && $mfaRequired ) { ?>
+
+                            <?php if ($mfaType === MfaUtils::TOTP) { ?>
```

Comment:
Would it not be better to set any required variables in controller instead of calling method on form render?

Reply:
Do you mean to change MfaUtils::TOTP to regular variable that created in the AuthorizationController?

Reply:
yep just to keep form consistent

Reply:
Done